packages/protocol/thread/thread.ts

```diff
@@ -518,12 +517,13 @@ class _ThreadFront {
   async getHitCounts(
     sourceId: SourceId,
     locations: SameLineSourceLocations[],
-    focusRegion: FocusRegion | null
+    beginExecutionPoint?: ExecutionPoint,
+    endExecutionPoint?: ExecutionPoint
```

Comment:
It might be better to make a `range` param here, because that is what we will ultimately call the protocol with, and passing in just one of these would be veering towards "hard to define" behavior.

Reply:
Sure!